drivers/spi/spi_ll_stm32.c

```diff
@@ -938,6 +938,15 @@ static void spi_stm32_irq_config_func_##id(const struct device *dev)		\
 #define STM32_SPI_USE_SUBGHZSPI_NSS_CONFIG(id)
 #endif
 
+#if !defined(CONFIG_SOC_SERIES_STM32F1X) \
+	&& (!defined(CONFIG_SOC_SERIES_STM32L1X) || defined(SPI_CR2_FRF))
```

Comment:
Rather than silently converting property to 0 for these cases, it should rather be notified at binding level,
and then generate an error if an unsupported case is configured

Reply:
We should generate an error if TI format selected but not supported